tests/app/test_state.py

```diff
@@ -0,0 +1,21 @@
+"""Test for ENPState Module."""
+
+from app.providers.provider_aws import ProviderAWS
+from app.state import ENPState
+
+
+def test_enp_state_initialization() -> None:
+    """Test to make sure ENPState can have provider attribute."""
+    state = ENPState()
+    state.providers = {'aws': ProviderAWS()}
```

Comment:
Did you intend `==`?
```suggestion
    assert isinstance(state.providers, dict)
```

Reply:
This makes more sense. Updated.